src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div id="agreement-summary" v-if="isSummary">
+      <v-card flat>
+        <!-- Summary Header -->
+        <div class="agreement-summary-header">
+            <v-icon>mdi-handshake</v-icon>
+            <label class="agreement-summary-title"><strong>Incorporation Agreement and Benefit </strong></label>
+        </div>
+
+        <!-- Summary Warning -->
+        <div v-if="showErrorSummary" class="agreement-invalid-message">
+            <span>
+            <v-icon color="#1976d2">mdi-information-outline</v-icon>
+            This step is not complete.
+            <router-link id="router-link" :to="{ path: '/incorporation-agreement' }">
+                Return to this step to complete it.
+            </router-link>
+            </span>
+        </div>
+        <!-- Summary Content -->
+        <div v-else class="summary-desc">
+            <div><v-icon color="green" class="agreement-valid-icon">mdi-check</v-icon></div>
+             <div>
+                {{ selectedAgreementDescription }}
+            </div>
+        </div>
+      </v-card>
+    </div>
+    <div v-else class="m1-1">
```

Comment:
I think you want "ml-4" (margin-left) here. See prev link for mapping to px.

Reply:
ml-4 looks weird. I tried that. This one looks aligned

Reply:
Weird.
You had `style="margin-left:1rem;"` before. The Vuetify class is exactly that, along with "!important".
"m1-1" is not a valid class.
Is it possible you don't need any margin style on this div?

Reply:
You are right. No need of margin there